src/views/pages/Swap/SwapPair.js

```diff
@@ -36,7 +36,7 @@ const SwapPair = ({ assetSwap, finalLpArray, web3 }) => {
     <>
       <Card className="card-body">
         <Row className="my-2">
-          <Col xs="6" className="">
+          <Col xs="auto">
```

Comment:
Changed the left-text Cols (labels?) to xs='auto'
Removed the 'xs=' from the right-text Cols
(Just ensure the left-text/labels arent changed to be long)